backend-rust/src/graphql_api/baker.rs

```diff
@@ -115,8 +115,32 @@ impl QueryBaker {
             }
             BakerSort::BakerApy30DaysDesc => todo!(),
             BakerSort::DelegatorApy30DaysDesc => todo!(),
-            BakerSort::BlockCommissionsAsc => todo!(),
-            BakerSort::BlockCommissionsDesc => todo!(),
+            BakerSort::BlockCommissionsAsc => {
+                Baker::block_commission_asc_connection(
```

Comment:
Fine that we implement it for compatibility, but I do not see any use for this particular feature.

Reply:
I kind of agree, but when it was not that hard to implement, and it is isolated in this function, so it would be easy to remove again.